### PR TITLE
[Remove Stream/Optional usage] LPS-174586 module blogs-service

### PR DESCRIPTION
**Pull request by JanisZhang**
https://issues.liferay.com/browse/LPS-174586

### CODE REVIEW

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by zhangyan0701** [comment]
ci:test:blogs

**Comment by catarineaguiar** [comment]
no regressions found.

File: modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -1876,13 +1875,14 @@ private boolean _isValidImageMimeType(FileEntry fileEntry) {
...
-				_blogsFileUploadsConfiguration.imageExtensions()
-			).anyMatch(
-				extension ->
-					extension.equals(StringPool.STAR) ||
-					extensions.contains(extension)
-			)) {
+		for (String extension :
+				_blogsFileUploadsConfiguration.imageExtensions()) {
+
+			if (!(extension.equals(StringPool.STAR) ||
```
Comment (dantewang) [inline modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/service/impl/BlogsEntryLocalServiceImpl.java:1881]: @JanisZhang You can just flip the condition here and `return true` in the if-body. It's a very simple one, no need to do shortcut.

**Review by dantewang — CHANGES_REQUESTED** [review]
Please see my comment and resend.

**Pull request closed without merging**